### ISSUE TITLE
Drawing pad shortcuts not working

### ISSUE THREAD

**Issue by NandorKovacs** [flxzt/rnote#857]
**Describe the bug**  
Drawing Pad Button 1-4 Action doesn't work on my Ubuntu

**To Reproduce**  
1. Set the Drawing Pad Button 1 Action to Eraser
2. Click the button
3. Click all other buttons, because what if im pushing the wrong button
4. Nothing happens

**Expected behavior**  
I would expect my tool to change to the tool that i configured it to change to, for example the eraser in the above description

**Console Output**  
2023-10-30T22:13:09.183Z DEBUG rnote > ... env_logger initialized

(rnote:2): Gtk-CRITICAL **: 23:13:11.223: gtk_widget_class_set_accessible_role: assertion '!gtk_accessible_role_is_abstract (accessible_role)' failed

**Desktop (please complete the following information):**  
 - OS: Ubuntu 23.04 x86_64
 - App Version: Rnote Version 0.9.0
 - Installation Source: Flatpak
 - Desktop Environment: Gnome 44.3

**Additional context**  
I am using a Wacom Intuos S 4100WL. In the settings, the tablet is set to Tablet Mode, and the buttons are mapped to "App defined" -> not to a keybind, or the on screen help, or to switch monitors, which would be the other options. 

It might be related to this pr #533

**Comment by flxzt** [comment]
Does the app log anything when one of the buttons is pressed?

**Comment by NandorKovacs** [comment]
Im unsure, i ran rnote with the command provided in the bug report form, and the terminal only printed what i copied into the issue. Is the debug output located in a file somewhere, or was there just nothing else?

**Comment by flxzt** [comment]
no, it should log a message [code here](https://github.com/flxzt/rnote/blob/0aef3ca2c8ea799cac1a7d85f45abc3ecb1c667e/crates/rnote-ui/src/appwindow/appwindowactions.rs#L357). Then the actions themselves are not triggered. I am not sure why.

**Comment by UlyssesZh** [comment]
Maybe related: flatpak/flatpak#4910

**Comment by flxzt** [comment]
Not sure about that, it seems to be only about electron apps which Rnote is not.

However, [this issue](https://gitlab.gnome.org/GNOME/mutter/-/merge_requests/3393) might be related. Gnome 45 apparently had broken tablet support. It's unclear if Gnome 44 also was affected, but it might be worth it to wait for a release that includes this fix

**Comment by abd-liac** [comment]
I have the very same issue on i3wm. From Opentabletdriver side, I can observe the state change when button pressed, but `RUST_LOG=rnote=debug RUST_BACKTRACE=1 rnote` output nothing relevant, IIUC.

```
 2023-12-03T16:26:08.055Z DEBUG rnote > ... env_logger initialized

(rnote:7309): Gtk-CRITICAL **: 00:26:08.200: gtk_widget_class_set_accessible_role: assertion '!gtk_accessible_role_is_abstract (accessible_role)' failed
 2023-12-03T16:26:08.202Z DEBUG rnote::canvas > did not load `engine-config` from settings, was empty
 2023-12-03T16:26:36.915Z DEBUG rnote_engine::store > state has not changed, no need to record

(rnote:7309): Gtk-WARNING **: 00:26:37.050: Layout managers of type GtkBoxLayout do not implement the GtkLayoutManager::create_layout_child method

(rnote:7309): Gtk-CRITICAL **: 00:26:37.050: The layout manager of type GtkBoxLayout 0x55a9a5a0db20 does not create GtkLayoutChild instances
```
